sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -860,6 +860,29 @@ class JDBCSuite extends QueryTest
       Some(TimestampType))
   }
 
+  test("MsSqlServerDialect jdbc type mapping") {
```

Comment:
Okay. Since this is the same with `master`, we can ignore adding JIRA ID.

Reply:
Are you referring to "test("MsSqlServerDialect jdbc type mapping")".  The fix updated this function, did not create a new function. For the new function that i added, i have mentioned the JIRA ID.

Reply:
That's exactly what I mean, @shivsood .

Reply:
Thanks